site/en/docs/android/custom-tabs/guide-interactivity/index.md

```diff
@@ -192,6 +192,21 @@ CustomTabsIntent intent = new CustomTabsIntent.Builder()
     .setSecondaryToolbarViews(secondaryToolbarViews, clickableIds, toolbarPendingIntent)
     .build();
 ```
+## Add an action button to the toolbar
+
+To add an additional action button to the toolbar, use the intent extra `EXTRA_SHOW_ON_TOOLBAR`:
+
+`actionIntent.putExtra(ActionBroadcastReceiver.EXTRA_SHOW_ON_TOOLBAR, true);`
+
+If the action icon doesn't fit on the toolbar, it will display on the second toolbar.
+
+## Disable bookmarks and download buttons
+
+To disable the bookmarks and download buttons from the three dot menu, use the following intent extras:
+
+`actionIntent.putExtra(ActionBroadcastReceiver.EXTRA_DISABLE_BOOKMARKS_BUTTON, true);`
```

Comment:
`actionIntent` is used in the section above for a PendingIntent. In this case I assume it should be a customTabsIntent:

```
CustomTabsIntent customTabsIntent = new CustomTabsIntent.Builder()
        .build();
customTabsIntent.intent.putExtra(ActionBroadcastReceiver.EXTRA_DISABLE_BOOKMARKS_BUTTON, true);
```

or am I missing something here?

Reply:
I pulled that out of the example syntax because no one gave me any other context, and I figured the best way to get a complete answer was to do it wrong first and get corrected.